bool stop
OneMotorSpeed left
OneMotorSpeed right
float32 duration_ms
---
bool result